File: Java/src/Student.java
public class Student 
{
    private String Name;
    private int Age;
    private String Address;
    
    public Student(String name, int age, String address) 
    {
        Name = name;
        Age = age;
        Address = address;
    }

    public String GetName() 
    {
        return Name;
    }

    public int GetAge() 
    {
        return Age;
    }

    public String GetAddress() 
    {
        return Address;
    }

    public String toString() 
    {
        return "(Name=" + Name + ", Age=" + Age + ", Address=" + Address + ")";
    }
}

File: C#/Student.cs
namespace CSharp;

public class Student
{
    public string Name { get; }
    public int Age { get; }
    public string Address { get; }

    public Student(string name, int age, string address)
    {
        Name = name;
        Age = age;
        Address = address;
    }

    public override string ToString()
    {
        return $"(Naam: {Name}, Leeftijd: {Age}, Adres: {Address})";
    }
}